apache/docker-entrypoint.sh

```diff
@@ -195,6 +195,7 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
 
   if [ ! -z "${ROUNDCUBEMAIL_ASPELL_DICTS}" ]; then
     ASPELL_PACKAGES=`echo -n "aspell-${ROUNDCUBEMAIL_ASPELL_DICTS}" | sed -E "s/[, ]+/ aspell-/g"`
+    which apt-get && apt-get update
     which apt-get && apt-get install -y $ASPELL_PACKAGES
```

Comment:
Could you pull those two lines together, so all Debian-specifics stay on one line?

```suggestion
    which apt-get && apt-get update && apt-get install -y $ASPELL_PACKAGES
```

Reply:
certainly.

should I also mute the output of `which`?